aldryn_newsblog/models.py

```diff
@@ -137,7 +149,7 @@ class Article(TranslatedAutoSlugifyMixin,
     tags = TaggableManager(blank=True)
 
     related = SortedManyToManyField('self', verbose_name=_('related articles'),
-                                    blank=True)
+                                    blank=True, symmetrical=ARTICLE_RELATED_SYMMETRICAL)
```

Comment:
This is a problem, because depending on your settings it will cause django to detect that a migration is necessary, showing a nasty warning. Does this need to be a setting?